agent-local/mysql

```diff
@@ -270,18 +270,18 @@ function ss_get_mysql_stats( $options ) {
    # hostname.
    $host_str  = $host.($port != 3306 ? ":$port" : '');
    debug(array('connecting to', $host_str, $user, $pass));
-   if (!extension_loaded('mysql') ) {
+   if (!extension_loaded('mysqli') ) {
       debug("The MySQL extension is not loaded");
       die("The MySQL extension is not loaded");
    }
    if ($mysql_ssl || (isset($options['mysql_ssl']) && $options['mysql_ssl']) ) {
-      $conn = mysql_connect($host_str, $user, $pass, true, MYSQL_CLIENT_SSL);
+      $conn = mysqli_connect($host_str, $user, $pass, true, MYSQL_CLIENT_SSL);
    }
    else {
-      $conn = mysql_connect($host_str, $user, $pass);
+      $conn = mysqli_connect($host_str, $user, $pass);
    }
    if (!$conn ) {
-      die("MySQL: " . mysql_error());
+      die("MySQL: " . $conn->connect_error);
```

Comment:
Good catch! My mistake. Checking out this, http://php.net/manual/en/mysqli.error.php, and the code there,
if ($mysqli->connect_errno) {
    printf("Connect failed: %s\n", $mysqli->connect_error);
    exit();
}

Seems the code above should be as follows, agreed?
if ($conn->connect_errno) {

I admit - I had it running, but missed checking with MySQL down. Again, thanks!